selfdrive/controls/controlsd.py

```diff
@@ -55,6 +55,8 @@ def __init__(self, sm=None, pm=None, can_sock=None):
       self.sm = messaging.SubMaster(['thermal', 'health', 'model', 'liveCalibration', 'ubloxRaw',
                                      'dMonitoringState', 'plan', 'pathPlan', 'liveLocationKalman',
                                      'frame', 'frontFrame'])
+      if SIMULATION:
+        self.sm.ignore_alive = ['ubloxRaw','frontFrame']
```

Comment:
Setting it right on the `__init__` like so? Worked as expected here, saved doing an `if` on every call to `update_events()`

Reply:
pass it in on instantiation